waspc/data/Generator/templates/sdk/wasp/server/config.ts

```diff
@@ -1,9 +1,9 @@
 {{={= =}=}}
 import merge from 'lodash.merge'
 
-import { stripTrailingSlash } from "wasp/universal/url";
+import { stripTrailingSlash } from "../universal/url.js";
 
-const env = process.env.NODE_ENV || 'development'
+const nodeEnv = process.env.NODE_ENV ?? 'development'
```

Comment:
I wouldn't discuss this bit here since it will change with #2362 PR - it says `NODE_ENV` is an enum type which can be either `development` or `production`.

This was just me cleaning up as I went and in hindsight, I probably should revert this - but it won't matter after the Zod PR.